doc/ale.txt

```diff
@@ -31,6 +31,8 @@ CONTENTS                                                         *ale-contents*
       erlc................................|ale-erlang-erlc|
     fortran...............................|ale-fortran-options|
       gcc.................................|ale-fortran-gcc|
+    handlebars............................|ale-handlebars-options|
+      ember-template-lint.................|ale-handlebars-embertemplatelint|
```

Comment:
I might rename the reference here later. The name is probably too long to fit in the table of contents. I can set marks for this and an abbreviation too.